extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -159,8 +161,16 @@ export class AmpGeo extends AMP.BaseElement {
    * @return {Promise}
    */
   findCountry_(ampdoc) {
+    const bodyElem = ampdoc.getBody();
+    const docElem = ampdoc.getRootNode().documentElement;
```

Comment:
`docElement` will be undefined in Shadow mode.

Reply:
could i just do bodyElem.parentElement in all cases to get `html` ?

Reply:
It's a direct child of the ShadowRoot, show `parentElement` will be null. I think we just need to handle the fact there's no guarantee of a `<html>` node, it maybe be nullish, and adjust the code that needs it accordingly.